janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -275,6 +275,8 @@ public boolean apply(Integer input) {
     private final boolean skipSchemaCheck;
     private final String compatClass;
     private final HBaseCompat compat;
+    // Cached return value of getDeployment() as requesting it can be expensive.
```

Comment:
OK. I don't want to block this PR. It does seem like the value will not change for BigTable. So, lets make an issue and see if creating a BigTableStoreManger as a subclass of HBaseStoreManager makes sense down the road!